pypeit/spectrographs/vlt_fors.py

```diff
@@ -331,3 +353,61 @@ def configuration_keys(self):
         return ['dispname', 'dispangle', 'decker', 'detector']
 
 
+    def parse_dither_pattern(self, file_list, ext=None):
```

Comment:
This should be generalized, i.e. made accessible to other spectrographs.

Reply:
Well it is pretty spectrograph specific, but I agree in general.